bot/bot.go

```diff
@@ -64,8 +64,8 @@ func (me *Bot) Monitor(trade config.Trade) error {
 	}
 
 	if currentPrice <= trade.BuyPrice {
-		me.Log.Debug().Msgf("[%s] current price=%.2f - nice to buy!",
-			trade.Symbol, currentPrice, trade.BuyPrice)
+		me.Log.Debug().Msgf("[%s] current price=%.3f - nice to buy!",
```

Comment:
I think it was a great idea, however i remembered that i'm trading some coins with 5 decimals, what do you think about increasing it to 5?

Reply:
I did it in this PR https://github.com/rafaelbmateus/go-binance-bot/pull/14